pkg/controller/builder/builder_component.go

```diff
@@ -20,11 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 package builder
 
 import (
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/types"
-
 	appsv1alpha1 "github.com/apecloud/kubeblocks/apis/apps/v1alpha1"
-	workloads "github.com/apecloud/kubeblocks/apis/workloads/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
ditto

Reply:
fixed in: [851aed9](https://github.com/apecloud/kubeblocks/pull/6776/commits/851aed915d2d9441ac101643776390a6417f5ad1)